### ISSUE TITLE
Accessibility - Fix Page Number Missing Label

### ISSUE THREAD

**Issue by vkoves** [vkoves/electrify-chicago#20]
Caught by Lighthouse, here's a screenshot from it:

![Screenshot from 2023-05-24 12-59-28](https://github.com/vkoves/electrify-chicago/assets/3187531/1dc4dc69-31d0-48c3-b9f2-f9d490a55ab0)

**Comment by vkoves** [comment]
Resolved via PR #21
